gef-ui/server/controllers/activities-controller/index.test.js

```diff
@@ -1,12 +1,25 @@
 import { fromUnixTime, format } from 'date-fns';
```

Comment:
Test should be `index.ff-test.ts`, please ensure all the unit test cases where FF is mocked then they are suffixed with `[file-name].ff-test.ts/.js` naming convention so they are executed in their respective jobs in CI-CD.